includes/html/print-interface.inc.php

```diff
@@ -72,7 +72,11 @@
 
 echo '</span>';
 
+$port_groups = [0 => 'Default'] + dbFetchKeyValue('SELECT id, name from port_groups order by name');
+
 echo "</td><td width=100 onclick=\"location.href='" . generate_port_url($port) . "'\" >";
+echo $port_groups[$port['port_group_id']];
+echo '</td><td width=100>';
```

Comment:
this doesnt really make sense, now if you click the port group it takes you to the port page, and when you click the port graph, nothing happens

Reply:
port group is not clickable ...

Reply:
`<td width=100 onclick=\"location.href='" . generate_port_url($port) . "'\" >` yes it is.

Reply:
oh yeah ... removed

Reply:
and now you cant click the graphs anymore